parse rpm-like tags: 1.2-3

--- semver-1.4.2/src/VersionParser.php	2016-08-30 19:08:34.000000000 +0300
+++ semver-1.4.2/src/VersionParser.php	2016-09-02 18:03:41.267807721 +0300
@@ -125,6 +125,11 @@
             $version = $match[1];
         }
 
+        // XXX: convert 1.2-3 to 1.2-p3
+        if (preg_match('/^(\d+[\d.]+)-(\d+)$/', $version, $m)) {
+            $version = "{$m[1]}-p{$m[2]}";
+        }
+
         // match classical versioning
         if (preg_match('{^v?(\d{1,5})(\.\d++)?(\.\d++)?(\.\d++)?' . self::$modifierRegex . '$}i', $version, $matches)) {
             $version = $matches[1]
